src/main/java/katas/Kata9.java

```diff
@@ -20,6 +19,26 @@
     public static List<Map> execute() {
         List<MovieList> movieLists = DataUtil.getMovieLists();
 
-        return ImmutableList.of(ImmutableMap.of("id", 5, "title", "some title", "time", new Date(), "url", "someUrl"));
+        //return ImmutableList.of(ImmutableMap.of("id", 5, "title", "some title", "time", new Date(), "url", "someUrl"));
+        return DataUtil.getMovieLists()
+                .stream()
+                .flatMap(movieList -> movieList.getVideos()
+                        .stream()
+                        .map(movie -> ImmutableMap.of(
+                                "id", movie.getId(),
+                                "title", movie.getTitle(),
+                                "time", movie.getInterestingMoments()
+                                        .stream()
+                                        .filter(interestingMoment -> interestingMoment.getType().equals("Middle"))
+                                        .map(InterestingMoment::getTime)
+                                        .findFirst()
+                                        .get()
+                                ,
+                                "url", movie.getBoxarts()
+                                        .stream()
+                                        .reduce((x1, x2) -> (x1.getHeight() * x1.getWidth()) > (x2.getHeight() * x2.getWidth()) ? x1 : x2)
+                                        .map(BoxArt::getUrl)
+                                        .get())))
```

Comment:
[mf] pls extract this huge block of code into separate private methods for better readability